src/vaults/vaults.utils.ts

```diff
@@ -555,10 +548,26 @@ export async function estimateVaultPerformance(
     throw new Error(`${vaultDefinition.name} does not have adequate harvest history`);
   }
 
-  const totalDuration = recentHarvests[0].timestamp - recentHarvests[data.length - 1].timestamp;
+  let totalDuration;
+
+  // TODO: generalize this for voting vaults + look up their voting periods
+  if (vaultDefinition.vaultToken === TOKENS.BVECVX) {
```

Comment:
Would `TOKENS.GRAVIAURA` be included in this as well, same voting period as cvx?